base_territory/models/res_state.py

```diff
@@ -0,0 +1,10 @@
+# Copyright (C) 2020 Open Source Integrators
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ResCountryState(models.Model):
+    _inherit = "res.country.state"
+
+    territory_id = fields.Many2one("res.territory", string="Territory")
```

Comment:
@murtuzasaleh Not needed. Can be removed.

Reply:
@max3903 Can I remove also `state_ids` One2many fields from `res_territory.py` file? https://github.com/OCA/field-service/pull/508/files#diff-859251c00fcc1e8c6bdeb87872dfcad7R22

Reply:
@murtuzasaleh Yes.